single-line-diagram-core/src/test/resources/InternalBranchesBusBreaker.svg

```diff
@@ -299,6 +299,7 @@ polyline {fill: none}
                 <circle cx="5" cy="15" r="5" transform="rotate(180.0,8.0,12.0)"/>
                 <circle cx="11" cy="15" r="5" transform="rotate(180.0,8.0,12.0)"/>
                 <circle cx="8" cy="19" r="5" transform="rotate(180.0,8.0,12.0)"/>
+                <text class="sld-label" id="T3_12_T3_12_NW_LABEL" x="-5.0" y="-5.0">T3_12</text>
```

Comment:
In fact adding such a label is the goal of current PR, but I think we don't want that for the default behaviour, therefore the `DefaultDiagramLabelProvider::getNodeLabels` should not add anything for the fictitious nodes